### DISCUSSION TITLE
Turbo Repo Compatibility

### DISCUSSION THREAD

**Discussion by AdeOlo** [obytes/react-native-template-obytes#391] [Q&A]
# Summary:
I am planning on creating Turbo Repo with Nextjs  for my web application and I wanted to ask if you think the starter kit is compatible with the way TurboRepo works especially the Lint, Husky configurations ?

**Comment by yjose** [comment]
Hey @AdeOlo, I'm not sure if I can help here. I used Turbo Repo a while ago, and I'm not confident I can answer your question. You will likely need to update some configurations to make it work as expected with a monorepo architecture ( mainly eslint, husky should work without issues) , but I'm unsure if there will be any limitations.

I will convert this issue into a discussion
